applications/nrf5340_audio/src/bluetooth/le_audio_bis_gateway.c

```diff
@@ -168,24 +199,42 @@ static int adv_create(void)
 		LOG_ERR("Failed to set periodic advertising parameters (ret %d)", ret);
 		return ret;
 	}
-	ret = bt_audio_broadcast_source_get_id(broadcast_source, &broadcast_id);
-	if (ret) {
-		LOG_ERR("Unable to get broadcast ID: %d", ret);
-		return ret;
+
+	if (IS_ENABLED(CONFIG_BT_AUDIO_BROADCAST_ID_RANDOM)) {
+		ret = bt_audio_broadcast_source_get_id(broadcast_source, &broadcast_id);
+		if (ret) {
+			LOG_ERR("Unable to get broadcast ID: %d", ret);
+			return ret;
+		}
+	} else {
+		broadcast_id = CONFIG_BT_AUDIO_BROADCAST_ID_FIXED;
 	}
 
 	/* Setup extended advertising data */
 	net_buf_simple_add_le16(&ad_buf, BT_UUID_BROADCAST_AUDIO_VAL);
 	net_buf_simple_add_le24(&ad_buf, broadcast_id);
 
+	public_broadcast_features_set(&pba_features);
+
+	net_buf_simple_add_le16(&pba_buf, 0x1856);
```

Comment:
Everything related to pba_buf can be guarded by #if (CONFIG_AURACAST)?

Reply:
Same for pba_features?

Reply:
Will refactor a bit to isolate it in #if